__tests__/app.test.js

```diff
@@ -34,4 +43,26 @@ describe('/api/topics', ()    =>  {
     }); 
 });
 
-
+describe('/api', ()    =>  {
+    describe('GET', ()   =>   {
+        it('Status code 200', () =>  {
+            return request(app).get("/api").expect(200);
+        });
+        it('should respond with an accurate JSON object with avaliable endpoints', ()   => {
+            return request(app)
+            .get("/api")
+            .expect(200)
+            .then((response) => {
+            expect(response.body.endpoint).toEqual(endPoint);
+            });
+        });
+        it('400: should respond with appropriate message when invalid url', ()  =>   {
+            return request(app)
+            .get("/ap")
+            .expect(400)
+            .then((response) => {
+                expect(response.body.msg).toBe('Bad Request')
+            });
+        });
```

Comment:
fabulous intention here but this technically isn't a 'bad request'. The user has done something perfectly valid (requested a path on the server) it just doesn't exist so what other status code could we put here?